README.md

```diff
@@ -1,36 +1,124 @@
-- S3 bucket in region where the solution will be deployed
+### Usage
 
-## Usage
-1. Get the solution by cloning this repository:
+#### Prerequisites
 
-`$ git clone https://github.com/aws-samples/aws-dms-sql-server.git`
+To deploy the application you will require an AWS account. If you don’t already have an AWS account, 
+create one at <https://aws.amazon.com> by following the on-screen instructions. 
+Your access to the AWS account must have IAM permissions to launch AWS CloudFormation templates that create IAM roles.
 
-2. Create S3 bucket: 
+#### Deployment
 
-`$ aws s3 mb s3://<BUCKET_NAME>/`
+The application is deployed as an [AWS CloudFormation](https://aws.amazon.com/cloudformation) template.
 
-3. Create a `.env` file and populate it with your own values:
+> **Note**  
+You are responsible for the cost of the AWS services used while running this sample deployment. There is no additional cost for using this sample. For full details, see the pricing pages for each AWS service you will be using in this sample. Prices are subject to change.
 
-`$ cp main/.env.example main/.env`
+1. Deploy the latest CloudFormation template by following the link below for your preferred AWS region:
 
-4. Run the deployment script: 
+|Region|Launch Template|
+|------|---------------|
+|**US East (N. Virginia)** (us-east-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-us-east-1/aws-dms-sql-server/v0.1/main.template)|
+|**EU (Ireland)** (eu-west-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-eu-west-1/aws-dms-sql-server/v0.1/main.template)|
 
-`$ env $(cat  main/.env | xargs) ./main/deploy.sh`
+2. If prompted, login using your AWS account credentials.
+1. You should see a screen titled "*Create Stack*" at the "*Specify template*" step. The fields specifying the CloudFormation 
+template are pre-populated. Click the *Next* button at the bottom of the page.
+1. On the "*Specify stack details*" screen you may customize the following parameters of the CloudFormation stack:
 
+|Parameter label|Default|Description|
+|---------------|-------|-----------|
+|Availability Zones|Requires input|The list of Availability Zones to use for the subnets in the VPCs. *Use two AZs*.|
+|On premise CIDR IP|Requires input|The CIDR Allowed RDP and SQL access to the EC2 and RDS host. CIDR block parameter must be in the form x.x.x.x/0-32.|
+|EC2 instance type|m5.2xlarge|The EC2 instance type for Microsoft SQL server.|
+|Windows server AMI|/aws/service/ami-windows-latest/Windows_Server-2019-English-Full-SQL_2016_SP2_Standard|The latest Windows server 2019 with SQL 2016 Standard AMI.|
```

Comment:
Can we put a link in the docs here where to find all the AMIs list?

Reply:
sure...Done